src/admin.ts

```diff
@@ -20,4 +56,37 @@ export class Admin {
     );
     return response.data.token as string;
   }
+
+  async getAvailableFunctions({ customerId, namespace }: Runnable) {
```

Comment:
Actually, could we call this just `getFunctions`?

Reply:
Also, I realized that we can't type the args here as just `Runnable`, since that'd for the user to supply the function name, etc. that wouldn't make sense here. We can add a new type for just the customerId/namespace pair!